packages/ember-metal/lib/map.js

```diff
@@ -182,7 +182,7 @@ OrderedSet.prototype = {
     @param {Function} fn
     @param self
   */
-  forEach: function(fn /*, thisArg*/) {
+  forEach: function(fn, ...thisArg) {
```

Comment:
@chadhietala so in this case, we are actually now allocating a new array, but before this PR its was merely a single index lookup. This (unless babel improves this) would be somewhat slower, but many extra bytes.

http://babeljs.io/repl/#?experimental=true&playground=false&evaluate=true&loose=false&spec=false&code=function%20before(b%2F*...foo*%2F)%20%7B%0A%20%20var%20a%20%3D%20arguments%5B1%5D%3B%0A%7D%0A%20%20%0Afunction%20after(b%2C%20...foo)%20%7B%0A%20%20%20var%20a%20%3D%20foo%5B1%5D%3B%0A%7D